src/AzExtWrapper.ts

```diff
@@ -3,11 +3,11 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzExtResourceType, IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
+import { IAzureQuickPickItem } from "@microsoft/vscode-azext-utils";
 import { AzureExtensionApiProvider } from "@microsoft/vscode-azext-utils/api";
 import { AppResource } from "@microsoft/vscode-azext-utils/hostapi";
-import { AzureResource } from '@microsoft/vscode-azext-utils/hostapi.v2';
 import { commands, Extension, extensions } from "vscode";
+import { AzExtResourceType, AzureResource } from '../api/src/index';
```

Comment:
Relative imports instead of package imports?

Reply:
I know it's a bit weird, but that was a choice I made on purpose. I want the code to be a native part of the resource groups extension, that also happens to be published as a package. This is a pretty common pattern for monorepos.

I wasn't so happy with having to import from a different "project" like this, so I did create an alternative:

Instead of putting the API code into a different folder from the extension code, I just put it in a folder within the extension src. And then you can easily configure a TS Config file to target only that folder and it builds only that folder. It works exactly the same, except the code lives in the same src directory.

Reply:
That's reasonable. I'll take a look at #532 and see what looks best.

Reply:
![image](https://user-images.githubusercontent.com/36966225/214945078-d7dd28f8-add5-419b-9081-c3b584defb5e.png)

Reply:
I think I'll just stick with this pattern